kotlin/samples/hello-terminal/app/src/main/java/com/squareup/sample/helloterminal/HelloWorkflow.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.sample.helloterminal
+
+import com.squareup.sample.helloterminal.HelloWorkflow.State
+import com.squareup.sample.helloterminal.terminalworkflow.ExitCode
+import com.squareup.sample.helloterminal.terminalworkflow.KeyStroke.KeyType.Backspace
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalConfig
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalRendering
+import com.squareup.workflow.RenderContext
+import com.squareup.workflow.Snapshot
+import com.squareup.workflow.StatefulWorkflow
+import com.squareup.workflow.WorkflowAction.Companion.emitOutput
+import com.squareup.workflow.WorkflowAction.Companion.enterState
+import com.squareup.workflow.WorkflowAction.Companion.noop
+import com.squareup.workflow.renderChild
+
+class HelloWorkflow : StatefulWorkflow<TerminalConfig, State, ExitCode, TerminalRendering>() {
```

Comment:
Any name but this. See `typealias` comment on `TerminalWorkflow`.

Reply:
But we already have

https://github.com/square/workflow/blob/79ac0fb418c18c8894a54d8faf3376efa5d0596d/kotlin/samples/hello-workflow/src/main/java/com/squareup/sample/helloworkflow/HelloWorkflow.kt#L13

Reply:
That's kind of my point. Tried to get that across…somewhere. God, I hate this disappearing and re-appearing reply UI.